### PR TITLE
Profile amittri1025 added

### PR DESCRIPTION
**Pull request by amittri1025**
Profile amitri1025 in json format

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/297"></a>

### CODE REVIEW

File: public/data/amittri1025.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Amit Tripathi",
+  "bio": "Web Developer | Passionate Learner | Open Source Enthusiast",
+  "avatar": "https://github.com/amittri1025.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/amitri1025",
```
Comment (rohan-kulkarni-25) [inline public/data/amittri1025.json:8]: - Please check github profile url once

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: public/data/amittri1025.json (hunk 1 above)
Comment (schmelto) [inline public/data/amittri1025.json:8]:
```suggestion
      "url": "https://github.com/amittri1025",
```

**Comment by amittri1025** [comment]
@schmelto  I corrected the suggestion and don't know how to mention that here

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**